test/gpu/native/compilerDriver.chpl

```diff
@@ -0,0 +1,8 @@
+config const N=10;
+
+on here.gpus[0] {
+  var A : [0..<N] int;
+  foreach i in 0..<N do
```

Comment:
Could you put an `@assertOnGpu` here to make sure we're codegen-ing for GPUs correctly with the compiler driver?

Reply:
Good idea, will do